src/v/redpanda/admin/api-doc/migration.def.json

```diff
@@ -169,6 +169,14 @@
         "migration": {
             "type": "outbound_migration",
             "$ref": "outbound_migration"
+        },
+        "created_timestamp": {
+            "type": "long",
+            "description": "time created, ms since epoch"
```

Comment:
This is going to go in docs, so should be a bit less cryptic. 
Is this correct:
The time of creation in milliseconds.
If not, can you add more detail to clarify? thx